### ISSUE TITLE
@types/react-redux seems broken on TypeScript 2.7.2

### ISSUE THREAD

**Issue by sorsaffari** [DefinitelyTyped/DefinitelyTyped#27080]
- [x] I tried using the `@types/xxxx` package and had problems.
- [x] I tried using the latest stable version of tsc. https://www.npmjs.com/package/typescript
- [x] I have a question that is inappropriate for [StackOverflow](https://stackoverflow.com/).
Authors: @tkqubo @thasner @kenzierocks @clayne11 

running 
**$ typedoc --options typedoc.json --out docs src/**
returns:
```
Using TypeScript 2.7.2 from path/to/app/node_modules/typedoc/node_modules/typescript/lib
Error: path/to/app/node_modules/@types/react-redux/index.d.ts(68)
 ';' expected.
Error: path/to/app/node_modules/@types/react-redux/index.d.ts(69)
 Declaration or statement expected.
```

running
**$ npm list typescript react react-redux @types/react-redux typedoc**
returns:
```
├── @types/react-redux@6.0.3
├── react@16.4.1
├── react-redux@5.0.7
├─┬ typedoc@0.11.1
│ └── typescript@2.7.2
└── typescript@2.9.2

@types/react @16.4.2
```

**Comment by SuhairZain** [comment]
@mamsoudi It seems that the fix was pulled and released as 0.12.0. I'm unable to see if this package is using that updated version.

**Comment by orta** [comment]
Hi thread, we're moving DefinitelyTyped to use [GitHub Discussions](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/53377) for conversations the `@types` modules in DefinitelyTyped.

To help with the transition, we're closing all issues which haven't had activity in the last 6 months, which includes this issue. If you think closing this issue is a mistake, please pop into the [TypeScript Community Discord](https://discord.gg/typescript) and mention the issue in the `definitely-typed` channel.
